Content.Server/Chemistry/Components/ChemMasterComponent.cs

```diff
@@ -1,362 +1,32 @@
-                    labelComponent.CurrentLabel = label;
-
-                    var bufferSolution = BufferSolution.SplitSolution(actualVolume);
-                    var bottleSolution = solSys.EnsureSolution(bottle, "drink");
-
-                    solSys.TryAddSolution(bottle, bottleSolution, bufferSolution);
-
-                    //Try to give them the bottle
-                    handSys.PickupOrDrop(user, bottle);
-                }
-            }
-            else //Pills
-            {
-                var individualVolume = BufferSolution.TotalVolume / FixedPoint2.New(pillAmount);
-                if (individualVolume < FixedPoint2.New(1))
-                {
-                    user.PopupMessageCursor(Loc.GetString("chem-master-window-buffer-low-text"));
-                    return;
-                }
-
-                var actualVolume = FixedPoint2.Min(individualVolume, FixedPoint2.New(50));
-                for (int i = 0; i < pillAmount; i++)
-                {
-                    var pill = _entities.SpawnEntity("Pill", _entities.GetComponent<TransformComponent>(Owner).Coordinates);
-
-                    //Adding label
-                    LabelComponent labelComponent = pill.EnsureComponent<LabelComponent>();
-                    labelComponent.OriginalName = _entities.GetComponent<MetaDataComponent>(pill).EntityName;
-                    string val = _entities.GetComponent<MetaDataComponent>(pill).EntityName + $" ({label})";
-                    _entities.GetComponent<MetaDataComponent>(pill).EntityName = val;
-                    labelComponent.CurrentLabel = label;
-
-                    var bufferSolution = BufferSolution.SplitSolution(actualVolume);
-                    var pillSolution = EntitySystem.Get<SolutionContainerSystem>().EnsureSolution(pill, "food");
-                    solSys.TryAddSolution(pill, pillSolution, bufferSolution);
+        [DataField("pillType"), ViewVariables(VVAccess.ReadWrite)]
+        public uint PillType = 0;
 
-                    //Change pill Sprite component state
-                    if (!_entities.TryGetComponent(pill, out SpriteComponent? sprite))
-                    {
-                        return;
-                    }
-                    sprite?.LayerSetState(0, "pill" + _pillType);
+        [DataField("mode"), ViewVariables(VVAccess.ReadWrite)]
+        public ChemMasterMode Mode = ChemMasterMode.Transfer;
 
-                    //Try to give them the bottle
-                    handSys.PickupOrDrop(user, pill);
-                }
-            }
+        [DataField("pillProductionLimit"), ViewVariables(VVAccess.ReadWrite)]
+        public uint PillProductionLimit = default!;
 
-            if (_bufferSolution?.Contents.Count == 0)
-                _label = "";
+        [DataField("bottleProductionLimit"), ViewVariables(VVAccess.ReadWrite)]
+        public uint BottleProductionLimit = default!;
```

Comment:
these should either have sensible defaults (like what they're set to in yaml right now) or be marked required because having them default to 0 production doesn't seem like correct behavior